DetoxSync/DetoxSync/Utils/_DTXTimerTrampoline.m

```diff
@@ -86,8 +86,8 @@ - (BOOL)isDead
 
 - (void)dealloc
 {
-	[self untrack];
-	
+	NSLog(@"🤦‍♂️ trampoline dealloc: %@ (tracking: %d)", self, _tracking);
```

Comment:
(I added this to verify the assumption above. I've not seen this called with tracking YES).

Reply:
:+1: